packages/create-block/lib/templates/esnext/src/index.js.mustache

```diff
@@ -33,6 +33,8 @@ import save from './save';
  * @see https://developer.wordpress.org/block-editor/developers/block-api/#registering-a-block
  */
 registerBlockType( '{{namespace}}/{{slug}}', {
+	apiVersion: 2,
```

Comment:
It would be nice to add a comment describing what purpose this setting has. If a doc page exists, we should link as well.

Reply:
A page has not been created yet. Let's wait with this PR until it is. :)

Reply:
I used the link to the blog post with the dev note. We can update later.